keyvi/src/cpp/stringdistance/distance_matrix.h

```diff
@@ -80,7 +80,7 @@ final {
       TRACE("boundary check %d %d", minimum_rows, number_of_rows_);
       if (minimum_rows > number_of_rows_) {
         // increase to given capacity or at least 120% of old number of rows
-        int new_rows = std::max(minimum_rows, (number_of_rows_ * 6) / 5);
+        size_t new_rows = std::max(minimum_rows, (number_of_rows_ * 6) / 5);
```

Comment:
as you touch this line, can you please add `const` qualifier (it is read-only for the rest of code) ?